src/main/java/com/redhat/devtools/intellij/tektoncd/tkn/TknCli.java

```diff
@@ -479,13 +485,22 @@ public String startLastTask(String namespace, String task) throws IOException {
     }
 
     @Override
-    public void showLogsPipelineRun(String namespace, String pipelineRun) throws IOException {
-        ExecHelper.executeWithTerminal(project, Constants.TERMINAL_TITLE,false, envVars, command, "pipelinerun", "logs", pipelineRun, "-n", namespace);
+    public void showLogsPipelineRun(String namespace, String pipelineRun, boolean toEditor) throws IOException {
+        if (!toEditor) {
+            ExecHelper.executeWithTerminal(project, Constants.TERMINAL_TITLE,false, envVars, command, "pipelinerun", "logs", pipelineRun, "-n", namespace);
+        } else {
+            executeWithEditor(namespace + "-" + pipelineRun + ".log", envVars, command, "pipelinerun", "logs", pipelineRun, "-n", namespace);
```

Comment:
Not sure if this can happen but without a qualifier you will face collision between taskruns and pipelineruns